src/main/java/com/stablekernel/rxespresso/RxIdlingResource.java

```diff
@@ -93,13 +98,12 @@ public void registerIdleTransitionCallback(ResourceCallback resourceCallback) {
     @Override
     public <T> Observable.OnSubscribe<T> onSubscribeStart(Observable<? extends T> observableInstance,
                                                           final Observable.OnSubscribe<T> onSubscribe) {
+
         int activeSubscriptionCount = subscriptions.incrementAndGet();
+        onSubscribeList.put(new WeakReference<Observable.OnSubscribe>(onSubscribe), new Throwable());
```

Comment:
That was kinda the point. "In reality it'll never happen.... but if it does we don't want to keep it from GCing". Since we never actually use it we don't really care when it becomes null. We are just using it as a counter of subscribed things.